public/resources/core/js/controllers/product-offering.controller.js

```diff
@@ -51,6 +51,34 @@
         vm.showFilters = showFilters;
         vm.getElementsLength = getElementsLength;
         vm.setFilters = setFilters;
+        vm.searchInput = "";
+
+        // Fill searchInput array
+        vm.fillSearchInput = fillSearchInput;
+        function fillSearchInput() {
+            return searchVM.searchInput;
+        }
+
+        // Initialize the search input content
+        vm.initializeInput = initializeInput;
+        function initializeInput() {
+            if($state.params.body !== undefined)
+                vm.searchInput = $state.params.body;
+        }
+
+        // Returns the input content
+        vm.getSearchInputContent = getSearchInputContent;
+        function getSearchInputContent() {
+            // Returns the content of the search input
+            return vm.searchInput;
+        }
+
+        // Handle enter press event on input
+        $("#searchparams").keyup(function(event){
```

Comment:
To be more consistent with angular you could have used ng-keyup directive with a method making the click